main.py

```diff
@@ -142,6 +153,46 @@ def create_circuit(self):
             result = 'The request body mimetype is not application/json.'
             log.debug('create_circuit result %s %s', result, 415)
             raise UnsupportedMediaType(result)
+        else:
+            log.info('############ validate spec: ################')
+            #home_dir = os.getcwd()
+            #yaml_file = "/home_dir/kytos/mef_eline/openapi.yml"
+            yaml_file = "/src/kytos-mef-eline/openapi.yml"
+            spec_dict, spec_url = read_from_filename(yaml_file)
+            # If no exception is raised by validate_spec(), the spec is valid.
+            try:
+                spec_error = validate_spec(spec_dict)
+            except ValueError as err:
+                log.info("validate spec_dict error: %s" % err)
+                raise BadRequest(err)
+            else:
+                if spec_error:
+                    log.info("invalid spec error: %s" % spec_err)
+                    raise BadRequest(spec_err)
+                else:
+                    spec = create_spec(spec_dict)
+                    validator = RequestValidator(spec)
```

Comment:
Loading the validator can be done at napp startup time, to save resources on every request.